test/e2e-tests.sh

```diff
@@ -24,10 +24,12 @@ source $(dirname $0)/e2e-common.sh
 initialize $@
 
 header "Setting up environment"
-
 install_triggers_crd
 install_pipeline_crd
 
+header "Running yaml tests"
+$(dirname $0)/e2e-tests-yaml.sh || fail_test
```

Comment:
Let me know if I'm mistaken, but I think `fail_test` exits the script. I think it could be beneficial to show the results from both the yaml and regular e2e tests. So, can you please change this to store the failure in a variable, and then run `fail_test` after the other e2e tests have run as well (see [Pipelines](https://github.com/tektoncd/pipeline/blob/master/test/e2e-tests.sh#L42) for a reference).